test/functional/helpers.lua

```diff
@@ -290,6 +290,22 @@ local function expect(contents)
   return eq(dedent(contents), curbuf_contents())
 end
 
+local function rmdir(path)
+  if lfs.attributes(path, 'mode') ~= 'directory' then
+    return nil
+  end
+  for file in lfs.dir(path) do
+    if file == '.' or file == '..' then
+      goto continue
+    end
+    if not os.remove(path..'/'..file) then
```

Comment:
Either works for me, but it should probably throw an error so that the test fails.